TCAT.xcodeproj/project.pbxproj

```diff
@@ -1027,7 +963,7 @@
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks";
 				OTHER_SWIFT_FLAGS = "$(inherited) \"-D\" \"COCOAPODS\" -DDEBUG";
 				"OTHER_SWIFT_FLAGS[arch=*]" = "$(inherited) \"-D\" \"COCOAPODS\"";
-				PRODUCT_BUNDLE_IDENTIFIER = com.cornellappdev.tcat;
+				PRODUCT_BUNDLE_IDENTIFIER = com.1cornellappdev.tcat;
```

Comment:
What happened here?

Reply:
yeah @Omarrasheed make sure we don't change bundle. Was most likely caused bc that bundle already existed, and he didnt know walker login. Login w/ appdev info and switch the bundle back